### PR TITLE
Add cluster command-line flag

### PR DESCRIPTION
**Pull request by paulgb**

### CODE REVIEW

File: src/commands/spawn.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export default class Spawn extends Command {
     grace: Flags.integer({ char: 'g', description: 'optional grace period (in seconds) to wait after last connection is closed before shutting down container' }),
     port: Flags.integer({ char: 'p', description: 'optional port for jamsocket to proxy requests to (default is 8080)' }),
     tag: Flags.string({ char: 't', description: 'optional tag for the service to spawn (default is latest)' }),
+    cluster: Flags.string({ char: 'c', description: 'optional cluster to spawn onto (default is jamsocket.run)'}),
```
Comment (rolyatmax) [inline src/commands/spawn.ts]: Perhaps we should add `hidden: true` to this flag to hide it from the docs / help command - at least until we have customers who have more than one cluster?

Also, I suppose creating a spawn token should accept the same `cluster` parameter?

File: src/commands/spawn.ts [hunk 2]
```diff
@@ -21,6 +21,7 @@ export default class Spawn extends Command {
     grace: Flags.integer({ char: 'g', description: 'optional grace period (in seconds) to wait after last connection is closed before shutting down container' }),
     port: Flags.integer({ char: 'p', description: 'optional port for jamsocket to proxy requests to (default is 8080)' }),
```
Comment (rolyatmax) [inline src/commands/spawn.ts]: Not related to this PR, but perhaps we should also hide the `port` flag from docs / help here and in the create spawn token command.

File: src/commands/spawn.ts (hunk 1 above)
Comment (paulgb) [reply src/commands/spawn.ts]: Agreed re. hiding it. Before adding it to the spawn token, I'd like to think through whether it should be a property of the token, the account, or the service.

**Review by rolyatmax — APPROVED** [review]
LGTM!

File: src/commands/spawn.ts (hunk 1 above)
Comment (rolyatmax) [reply src/commands/spawn.ts]: If we end up deciding that it should be a property of either the account or the service, would there be any reason to pass it in here or would we remove this flag?

**Pull request merged**